tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/PingThread.java

```diff
@@ -84,15 +84,17 @@ protected void ping() {
 
 				URL pingUrl = new URL(url);
 				URLConnection conn = pingUrl.openConnection();
+
 				((HttpURLConnection)conn).setInstanceFollowRedirects(false);
-				((HttpURLConnection)conn).getResponseCode();
+				int code = ((HttpURLConnection)conn).getResponseCode();
```

Comment:
Changing this code makes me nervous.  Are you sure that the previous code was not working correctly?  I'm worried this could get more unstable.